frontend/src/components/password/PasswordAdornment.tsx

```diff
@@ -1,12 +1,18 @@
 import { EyeIcon, EyeOffIcon } from "lucide-react";
 import { useEffect, useState } from "react";
+import { cn } from "src/lib/utils";
 
 type Props = {
   onChange: (viewingPassword: boolean) => void;
   isRevealed?: boolean;
+  iconClass?: string;
 };
 
-export default function PasswordViewAdornment({ onChange, isRevealed }: Props) {
+export default function PasswordViewAdornment({
```

Comment:
Could you make the filename and component name match? would the component make more sense as `RevealPasswordToggle`?